modules/http/src/test/java/org/mule/module/http/internal/request/DefaultHttpRequesterTestCase.java

```diff
@@ -7,29 +7,72 @@
 package org.mule.module.http.internal.request;
 
 import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mule.module.http.api.HttpConstants.Protocols.HTTP;
+import static org.mule.module.http.api.HttpConstants.Protocols.HTTPS;
 import org.mule.api.lifecycle.InitialisationException;
+import org.mule.module.http.internal.domain.InputStreamHttpEntity;
+import org.mule.module.http.internal.domain.request.HttpRequest;
+import org.mule.module.http.internal.domain.request.HttpRequestAuthentication;
+import org.mule.module.http.internal.domain.response.HttpResponse;
 import org.mule.tck.junit4.AbstractMuleContextTestCase;
+import org.mule.util.ValueHolder;
 
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Spy;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.stubbing.Answer;
 
-
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Can't you just declare the spy inline?

Reply:
I think so but, isn't this the preferred way?